### DISCUSSION TITLE
Flowbite disrupting django admin

### DISCUSSION THREAD

**Discussion by Gichinga** [themesberg/flowbite#646] [Q&A]
flowbite is disrupting the styling of the django admin, how do i resolve this issue, I want the admin page styling to be the default one provided by django

**Comment by vanruc** [comment]
you may need to add static finder  like below:
STATICFILES_FINDERS = ('compressor.finders.CompressorFinder','django.contrib.staticfiles.finders.AppDirectoriesFinder')
